### ISSUE TITLE
$.sceditor.plugins.bbcode.bbcode.remove doesn't work (jQuery 3.3.1 issue?)

### ISSUE THREAD

**Issue by SP4NQ** [samclarke/SCEditor#736]
In a previous version of jQuery, the .remove() feature worked nicely, but now Chrome states it cannot read the property of 'bbcode'. I'm not sure if this is due to changes in newer jQuery builds, but I'm wondering if there's a way to make it work with 3.3.1.

My initialization is as follows:

$(function(){$("#comment").sceditor({autoUpdate:true,autoExpand:false,resizeWidth:false,enablePasteFiltering:true,format:"bbcode",plugins:"undo,pastetext",toolbar:"bold,italic,underline,strike|cut,copy,pastetext|bulletlist|quote|emoticon,image,youtube,email,link,unlink|removeformat,source",emoticonsRoot:"/scripts/sceditor/",style:"/scripts/sceditor/themes/content/default.min.css"});
$.sceditor.plugins.bbcode.bbcode.remove("size").remove("justify").remove("font").remove("color").remove("subscript").remove("superscript").remove("left").remove("center").remove("right").remove("orderedlist").remove("table").remove("horizontalrule").remove("rtl").remove("print");});

Everything else works fine except for .remove().

**Comment by brunoais** [comment]
Are you using the SCE editor with the jQuery plugin?
Which version of SCE are you using?
This shouldn't be a problem with jQuery itself.
What does this output: `console.log($.sceditor.plugins.bbcode.bbcode, "\n"); console.log($.sceditor.plugins.bbcode.bbcode.remove)` ?

**Comment by SP4NQ** [comment]
Yes. Using jquery.sceditor.bbcode.min.js. I'm running the last version of SCEditor, 2.1.3.

Console output with both lines results in:

> Uncaught TypeError: Cannot read property 'plugins' of undefined

The general error I get when the page and text editor are loaded is:

> jquery-3.3.1.min.js:2 jQuery.Deferred exception: Cannot read property 'bbcode' of undefined TypeError: Cannot read property 'bbcode' of undefined

I've used .remove() before without any issues, but this was a couple years back running both an older version of jQuery (1.10.2) and SCEditor (1.4.7). I'm actually using the same initialization code from back then. And, again, everything works great except for .remove(). I'm wanting to use it to remove certain BBCode tags that are applied to copy and paste functions, such as [font], [size], and [color].

**Comment by brunoais** [comment]
Then I don't know what the problem is, we will have to wait for Sam....
Whenever he comes back 🙁

**Comment by SP4NQ** [comment]
I'm by no means a JS expert, but looking between the two versions of SCEditor, 1.4.7 and 2.1.3, it looks like the newer one doesn't even have bbcode.bbcode support anymore. The functionality appears to be missing entirely or it's execution has changed.
